src/js/utils/option-utils.jsx

```diff
@@ -30,7 +30,9 @@ export const debounceUsersFetch = (waitTime, minSearchLength) =>
 export const debounceLocationsFetch = (waitTime, minSearchLength) =>
   _.debounce((searchTerm, callback) => {
     if (searchTerm && searchTerm.length >= minSearchLength) {
-      apiClient.get(`/openboxes/api/locations?direction=${queryString.parse(window.location.search).direction}&name=${searchTerm}`)
+      const { direction } = queryString.parse(window.location.search);
+      const { type } = queryString.parse(window.location.search);
```

Comment:
Why not passing type to this function basing on the workflow. Why you want to have it in url params?